### PR TITLE
issue template config changes

### PR DESCRIPTION
**Pull request by thecoder876**
if someone doesn't like that i set `false` in `blank_issues_enabled: false` to `true`, request changes
cc @retronbv, @GrahamSH-LLK, @wgyt

**Resolves**

resolves nothing

**Changes**

i made i few small edits to the issue template `config.yml`

**Reason for changes**

for convenience

**Tests**

no, but it obviously works

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-blank_issues_enabled: false
+blank_issues_enabled: true
```
Comment (retronbv) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Please no, we have a blank template

**Review by retronbv — CHANGES_REQUESTED** [review]

**Comment by wgyt** [comment]
it is fine as is

**Comment by thecoder876** [comment]
@wgyt ugh then i'll remove that

**Comment by thecoder876** [comment]
@wgyt WHY

**Pull request closed without merging**